src/layouts/MyLayout.vue

```diff
@@ -34,18 +35,15 @@
     >
       <q-scroll-area class="fit">
         <q-list padding>
-          <q-item v-for="link in links" :key="link.text" v-ripple clickable>
-            <q-item-section avatar>
-              <q-icon color="grey" :name="link.icon" />
-            </q-item-section>
-            <q-item-section>
-              <q-item-label>{{ link.text }}</q-item-label>
-            </q-item-section>
-          </q-item>
+          <q-option-group
```

Comment:
the way which was on Youtube was good. could we not use this options?

Reply:
will rework to as youtube